KLSadd_insertion/src/updateChapters.py

```diff
@@ -1,110 +1,367 @@
+
+                if temp in khyper_header_chap:
+                    try:
+                        chap[tempref[d + 1] - 1] += "\paragraph{\\bf KLS Addendum: " + word + "}"
+                        chap[tempref[d + 1] - 1] += k_hyper_sub_chap[k_hyp_index_iii + offset]
+                        if chap9 == 1:
+                            sorterIIIcheck[sortloc][1] += 1
+                        k_hyp_index_iii += 1
+                    except IndexError:
+                        print("Warning! Code has found an error involving section finding for '" + name_chap + "'.")
+        elif sep1 == 1:
+            if name_chap in line.lower() and "orthogonality relation" not in line:
+                hyper_subs_chap.append([tempref[d + 1]])  # appends the index for the line before following subsection
+                hyper_headers_chap.append(temp)  # appends the name of the section the hypergeo subsection is in
+
+                if temp in khyper_header_chap:
+                    try:
+                        chap[tempref[d + 1] - 1] += "\paragraph{\\bf KLS Addendum: " + word + "}"
+                        chap[tempref[d + 1] - 1] += k_hyper_sub_chap[k_hyp_index_iii + offset]
+                        k_hyp_index_iii += 1
+                    except IndexError:
+                        print("Warning! Code has found an error involving section finding for '" + name_chap + "'.")
+        elif sep1 == 2:
+            if "hypergeometric representation" in line.lower():
+                hyper_subs_chap.append([tempref[d + 1]])  # appends the index for the line before following subsection
+                hyper_headers_chap.append(
+                    temp)  # appends the name of the section the hypergeo subsection is in so we can compare
+
+                if temp in khyper_header_chap:
+                    try:
+                        chap[tempref[d + 1] - 1] += "\paragraph{\\bf KLS Addendum: Special Value }"
+                        chap[tempref[d + 1] - 1] += k_hyper_sub_chap[k_hyp_index_iii + offset]
+                        k_hyp_index_iii += 1
+                    except IndexError:
+                        print("Warning! Code has found an error involving section finding for '" + name_chap + "'.")
+
+    if len(hyper_headers_chap) != 0:
+        print "Stuff for checking"
+        print k_hyper_sub_chap
+        print khyper_header_chap
+        print hyper_headers_chap
+        print word
+        sortmatch.append(word)
+        sortmatch_2.append(k_hyper_sub_chap)
+        sorterIIIcheck[sortloc][2] = word
+
+
+def cutwords(word_to_find, word_to_search_in):
+    """
+    This function checks through the outputs of later sections and removes duplicates, so that the output file does
+    not have duplicates.
+
+    IN DEVELOPMENT
+
+    :param word_to_find: Word that is being found
+    :param word_to_search_in: The big word that is being searched
+    :return: The big word without the word that was searched for
+    """
+    a = word_to_find
```

Comment:
Use the verbose names throughout the function instead of renaming them to `a` and `b`